### PR TITLE
Show 6502 opcode corresponding to byte

### PR DESCRIPTION
**Pull request by breqdev**
image

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -90,6 +93,26 @@ function roll(val?: number) {
   charDescElem.textContent = getNameForByteChar(byte)
   hexElem.textContent = byte.toString(16).padStart(2, "0").toUpperCase()
 
+  const opcodeInfo = OPCODES[byte];
+
+  opcodeElem.appendChild(document.createElement("span")).textContent = opcodeInfo.name;
```
Comment (AlpacaFur) [inline src/main.ts:98]: I will be stealing this pattern >:) I'd never put two and two together that you could property assign off of the returned node but it makes sense!

File: src/main.ts [hunk 2]
```diff
@@ -90,6 +93,26 @@ function roll(val?: number) {
...
+
+  const descElem = document.createElement("span");
+  descElem.textContent = opcodeInfo.desc;
+  descElem.style.fontSize = "16px";
+  opcodeElem.appendChild(descElem);
+
+  const addressingElem = document.createElement("span");
+  addressingElem.textContent = opcodeInfo.addressing;
+  addressingElem.style.fontSize = "16px";
+  opcodeElem.appendChild(addressingElem);
```
Comment (AlpacaFur) [inline src/main.ts]: Can we shift the font-size here into the CSS?

Something like this should target the opcode spans:
```css
#opcode>span:not(:first-child) {
  font-size: 16px;
}
```

**Review by AlpacaFur — APPROVED** [review]
Looks great, thanks for the PR! I lowkey didn't expect people to have to engage with the codebase so sorry for the current state of the code 😭.

File: src/main.ts (hunk 1 above)
Comment (breqdev) [reply src/main.ts:98]: github copilot came up with it 🫣

File: src/main.ts (hunk 2 above)
Comment (breqdev) [reply src/main.ts]: tested and looks good to me! updated the diff

**Comment by AlpacaFur** [comment]
Awesome, you're all set to merge!

**Pull request merged**